<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWidgetIOTriggerItem</class>
 <widget class="QWidget" name="ListWidgetIOTriggerItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="TaskNameCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
  </widget>
  <widget class="QComboBox" name="IOTriggerPortCbB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>5</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="IOTriggerWayCbB">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>5</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
